docs/content/preview/explore/observability/prometheus-integration/macos.md

```diff
@@ -124,60 +120,43 @@ Paste the following expressions into the **Expression** box and click **Execute*
 
 ### Throughput
 
-> Read IOPS
+**Read IOPS**
 
 ```sh
 sum(irate(rpc_latency_count{server_type="yb_cqlserver", service_type="SQLProcessor", service_method="SelectStmt"}[1m]))
 ```
 
 ![Prometheus Read IOPS](/images/ce/prom-read-iops.png)
 
-> Write IOPS
+**Write IOPS**
 
 ```sh
 sum(irate(rpc_latency_count{server_type="yb_cqlserver", service_type="SQLProcessor", service_method="InsertStmt"}[1m]))
 ```
 
-![Prometheus Read IOPS](/images/ce/prom-write-iops.png)
+![Prometheus Write IOPS](/images/ce/prom-write-iops.png)
 
 ### Latency
 
-> Read Latency (in microseconds)
+**Read Latency (in microseconds)**
 
 ```sh
 avg(irate(rpc_latency_sum{server_type="yb_cqlserver", service_type="SQLProcessor", service_method="SelectStmt"}[1m])) /
 avg(irate(rpc_latency_count{server_type="yb_cqlserver", service_type="SQLProcessor", service_method="SelectStmt"}[1m]))
 ```
 
-![Prometheus Read IOPS](/images/ce/prom-read-latency.png)
+![Prometheus Read Latency](/images/ce/prom-read-latency.png)
 
-> Write Latency (in microseconds)
+**Write Latency (in microseconds)**
 
 ```sh
 avg(irate(rpc_latency_sum{server_type="yb_cqlserver", service_type="SQLProcessor", service_method="InsertStmt"}[1m])) /
 avg(irate(rpc_latency_count{server_type="yb_cqlserver", service_type="SQLProcessor", service_method="InsertStmt"}[1m]))
 ```
 
-![Prometheus Read IOPS](/images/ce/prom-write-latency.png)
-
-## Clean up (optional)
-
-Optionally, you can shut down a local cluster as follows:
-
-```sh
-./bin/yugabyted destroy \
-                  --base_dir=node-1
-```
-
-```sh
-./bin/yugabyted destroy \
-                  --base_dir=node-2
-```
+![Prometheus Write Latency](/images/ce/prom-write-latency.png)
 
-```sh
-./bin/yugabyted destroy \
-                  --base_dir=node-3
-```
+{{% explore-cleanup-local %}}
```

Comment:
just before what's next- You can mention something like "Skip clean up if you want to create dashboards using the Prometheus metrics", because grafana dashboards use the steps from the prometheus page.